src/v0/destinations/hs/util.js

```diff
@@ -223,7 +224,8 @@ const getTransformedJSON = async (message, destination, propertyMap) => {
       // lowercase and replace ' ' & '.' with '_'
       const hsSupportedKey = formatKey(traitsKey);
       if (!rawPayload[traitsKey] && propertyMap[hsSupportedKey]) {
-        let propValue = traits[traitsKey];
+        // HS accepts empty string to remove the property from contact
```

Comment:
Can you add some inline HS docs for this?

Reply:
added in corresponding linear card

Reply:
No I am saying add link to Hs docs in your code for future ref